app.js

```diff
@@ -20,7 +20,12 @@ const tracer = require('./tracer')(JAMBONES_OTEL_SERVICE_NAME);
 const api = require('@opentelemetry/api');
 srf.locals = {...srf.locals, otel: {tracer, api}};
 
-const opts = {level: JAMBONES_LOGLEVEL};
+const opts = {
+  level: JAMBONES_LOGLEVEL,
+  timestamp: () => {
```

Comment:
Hmm, interesting,.  Why is this needed? I removed calls to timestampt like this a while ago because there is a cost in terms of performance - probably only noticed at very high call loads, but still

Reply:
Hi Dave, about the timestamp, sometime I need to calculate how much time between 2 event, but yes, If it's impact the performance, I will remove it after finish debuging
maybe I will put a flag there to enable this feature instead of completely remove it


when debuging by log, timestamp is quite important to map the event with other tool like pcap and another server log

Reply:
yes, keep in mind when looking at a log file you can pipe it through pino-pretty to get timestamps, like this
```
cat ~/.pm2/log/jambonz-feature-server.log | pino-pretty -c -t 
```

Reply:
as now there is pino-pretty, I removed this code.